adsys.proto

```diff
@@ -12,12 +12,16 @@ service service {
   rpc DumpPoliciesDefinitions(DumpPolicyDefinitionsRequest) returns (stream DumpPolicyDefinitionsResponse);
   rpc GetDoc(GetDocRequest) returns (stream StringResponse);
   rpc ListDoc(ListDocRequest) returns (stream StringResponse);
-  rpc ListActiveUsers(Empty) returns (stream StringResponse);
+  rpc ListActiveUsers(ListActiveUsersRequest) returns (stream StringResponse);
   rpc GPOListScript(Empty) returns (stream StringResponse);
 }
 
 message Empty {}
 
+message ListActiveUsersRequest {
```

Comment:
I think "Active" will bring confusion there. I don’t mind if we change the API completely.

wdyt about:
`ListUsers` with a Request having: `active` as a bool?

So, by default, we show all users with a cache, and then, can filter active this way.